src/resources.py

```diff
@@ -114,6 +117,10 @@ def list_resources(label, label_value, target_folder, request_url, request_metho
     if script and files_changed:
         execute(script)
 
+    if request_ignore_initial_event and ignore_request:
+        logger.debug(f"Ignoring sending request for initial list {resource} {metadata.namespace}/{metadata.name}")
```

Comment:
At this point in the code `metadata` points to the last item of the loop. Which might be a different one than the item which made `ignore_request` flip to `True`.